packages/test-generated/spec/reflection-equals.spec.ts

```diff
@@ -76,34 +79,59 @@ describe('reflectionEquals()', function () {
         expect(reflectionEquals(info, a, c)).toBeTrue();
     });
 
-    describe('should return true for all cloned fixture messages', function () {
-        fixtures.usingMessages((typeName, key, msg) => {
-            it(`${typeName} '${key}'`, function () {
-                const mi = fixtures.makeMessageInfo(typeName);
-                const mt = new MessageType<UnknownMessage>(mi.typeName, mi.fields, mi.options);
-                let copy = reflectionCreate(mt);
-                reflectionMergePartial(mi, copy, msg);
-                let eq = reflectionEquals(mi, msg, copy);
-                expect(eq).toBeTrue();
-            });
-        });
+    it('oneof scalars are equal', () => {
+        const mi = {
+            typeName: OneofScalarMemberMessage.typeName,
+            fields: OneofScalarMemberMessage.fields.map(normalizeFieldInfo),
+            options: {}
+        };
+        const msg = {
+            result: {
+                oneofKind: "value",
+                value: 42
+            }
+        } as PartialMessage<OneofScalarMemberMessage>;
+        const mt = new MessageType<UnknownMessage>(mi.typeName, mi.fields, mi.options);
```

Comment:
The fixtures did not provide MesssageTypes, and we had to use mergePartial. But here, we do have generated message types, and reduce the test to just reflectionEquals():

```suggestion
        const make = (): OneofScalarMemberMessage => ({
            result: {
                oneofKind: "value",
                value: 42
            }
        });
        const eq = reflectionEquals(
            OneofScalarMemberMessage,
            make() as unknown as UnknownMessage,
            make() as unknown as UnknownMessage
        );
        expect(eq).toBeTrue();
```

Ok, we rely on the generated interface matching the generated field info, but that still seems simpler than the fixtures. WDYT?